external/libp2p-service.js

```diff
@@ -180,7 +180,7 @@ class Libp2pService {
                     )
                 } else {
                     await pipe(
-                        ['ack'],
+                        [constants.NETWORK_RESPONSES.ACK],
```

Comment:
We need to handle this better, it is not consistent now, let's discuss it in the next sprint